lib/onebox/engine.rb

```diff
@@ -42,7 +42,7 @@ def options=(opt)
 
     def initialize(url, timeout = nil)
       @errors = {}
-      @options = DEFAULT
+      @options = DEFAULT.dup
```

Comment:
What is the point of even having this in a const? Can't we just do `@options = {}`?

Reply:
Good point, let me remove it to see if tests fail. We don't need to keep backwards compatibility with this constant anymore.

Reply:
Cool thanks. Just seems really weird to me to even make this into a const in the first place!